backend/tournesol/admin.py

```diff
@@ -49,7 +49,7 @@ class EntityCriteriaScoreAdmin(admin.ModelAdmin):
 class ContributorRatingAdmin(admin.ModelAdmin):
```

Comment:
future considerations

we could add a filter on entity type and and filter on poll in this admin